data/wanderer/wanderers middle.txt

```diff
@@ -3071,9 +3071,7 @@ mission "Wanderers: Sestor: Exiles 3"
 	to offer
 		has "Wanderers: Sestor: Exiles 2: active"
 	on offer
-		log "Succeeded in communicating with the Korath exiles, by landing on the one habitable planet in their territory. (It seems like the exiles might have a taboo against using weapons on that world.) The exiles agreed to send some ships to help disable the remaining Kor Sestor factory."
 		fail "Wanderers: Sestor: Exiles 2"
```

Comment:
Move this to the on accept as well. The offer conditions of this mission require that Exiles 2 is active. By failing 2 when 3 is offered but allowing 3 to be deferred, we might as well be making the "Leave" choice decline. But given that the endpoint is defer rather than decline, the intention seems to be that you can keep coming back here until you decide to meet with the Korath. Therefore, this action also needs moved.

Reply:
ok got it